server.js

```diff
@@ -130,12 +132,47 @@ app.delete("/api/week/", async (req, res) => {
   }
 });
 
-// Serve any static files
+app.get("/api/shoppingList", async (req, res) => {
+  try {
+    const groceryList = await getShoppingListMongo();
+    if (!groceryList) {
+      res.status(404).send("Could not find any groceries");
+      return;
+    }
+    res.send(groceryList);
+  } catch (error) {
+    console.error(error);
+    res.status(500).send("An internal server error occured");
+  }
+});
+
+app.post("/api/shoppingList", async (req, res) => {
+  const shoppingItem = req.body;
+  try {
+    await addShoppingItemMongo(shoppingItem);
+    res.send(`Successfully inserted ${shoppingItem}`);
+  } catch (error) {
+    console.error(error);
+    res
+      .status(500)
+      .send("An unexpected error occured. Please try again later!");
+  }
+});
+
+app.post("/api/shoppingList/many", async (req, res) => {
+  const Ingredients = req.body;
+  try {
+    await insertIngredientsToShoppingMongo(Ingredients);
+    res.send(`Successfully inserted ${Ingredients}`);
+  } catch (error) {
+    console.error(error);
+    res
+      .status(500)
+      .send("An unexpected error occured. Please try again later!");
+  }
+});
```

Comment:
You can remove `/api/shoppingList/many` route if you extend this:

```suggestion
app.post("/api/shoppingItems", async (req, res) => {
  try {
    const newShoppingItems = Array.isArray(req.body) ? req.body : [req.body];
    const inserted = await insertShoppingItems(newShoppingItems);
    res.send(`Successfully inserted ${inserted} shopping items`);
  } catch (error) {
    console.error(error);
    res
      .status(500)
      .send("An unexpected error occured. Please try again later!");
  }
});
```